Formula/s-nail.rb

```diff
@@ -0,0 +1,44 @@
+class SNail < Formula
+  desc "Fork of Heirloom mailx."
+  homepage "https://www.sdaoden.eu/code.html"
+  url "https://www.sdaoden.eu/downloads/s-nail-14.9.3.tar.gz"
+  sha256 "9048abe94c8b732ddefcd70b9f6052da16977de76b07598790e4c763e97f911d"
+
+  # Define options
+  option "with-spamc", "Includes a client for Spamassasin"
+
+  # Formula depends on OpenSSL and IDNA
+  depends_on "openssl"
+  depends_on "libidn"
+
+  def install
+    args = %W[
+      OPT_AUTOCC=no
+      CC=clang
+      cc_maxtopt=1
+      OPT_NOMEMBDBG=1
+      C_INCLUDE_PATH=#{Formula["openssl"].opt_include}
+      LDFLAGS=-L#{Formula["openssl"].opt_lib}
+      VAL_PREFIX=#{prefix}
+      OPT_DOTLOCK=no
+    ]
+
+    # Add optional SpamC
+    args << "OPT_SPAM_SPAMC=yes" if build.with?("spamc")
+
+    # Somehow make doesn't like a file called INSTALL being present.
+    # File is included in the .PHONY variable in the makefile but nonetheless.
+    # Install will fail if not renamed/deleted.
```

Comment:
Has this been reported upstream? It will need to be fixed upstream before this can be included in Homebrew.